src/battle_util.c

```diff
@@ -7891,11 +7880,14 @@ static u16 CalcMoveBasePower(u16 move, u8 battlerAtk, u8 battlerDef)
         #endif
         break;
     }
+    case EFFECT_GRAV_APPLE:
+        if (gFieldStatuses & STATUS_FIELD_GRAVITY)
+            MulModifier(&basePower, UQ_4_12(1.5));
+        break;
     case EFFECT_BEAT_UP:
         #if B_BEAT_UP_DMG >= GEN_5
         basePower = CalcBeatUpPower();
         #endif
-        break;
```

Comment:
This `break;` should probably be added back.

Reply:
Done!